testing/log/service.go

```diff
@@ -355,12 +355,18 @@ func (s *service) listen(context *endly.Context, request *ListenRequest) (*Liste
 			return nil, fmt.Errorf("listener has been already register for %v", logType.Name)
 		}
 	}
-	service, err := storage.NewServiceForURL(source.URL, source.Credentials)
+
+	fs, err := estorage.StorageService(context, source)
+	if err != nil {
+		return nil, err
+	}
+	source, storageOpts, err := estorage.GetResourceWithOptions(context, source)
```

Comment:
you can use err from `fs, err := estorage.StorageService(context, source)` instead of creating new one